girderformindlogger/api/v1/user.py

```diff
@@ -683,6 +683,12 @@ def getUsersDetails(self):
                required=False, dataType='boolean')
         .param('status', 'The account status (admin access required)',
                required=False, enum=('pending', 'enabled', 'disabled'))
+        .param(
+             'login',
+             'Deprecated. Do not use.',
```

Comment:
Why add a new, immediately deprecated parameter to a deprecated API call rather than creating a new call?